.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+ECJ_VERSION="4.6.1"
```

Comment:
@Nimfadora , 
maven central is stale ... last artifact is of 2016 year.
according to https://bugs.eclipse.org/bugs/show_bug.cgi?id=499019
location that Eclipse team use is at https://repo.eclipse.org/content/repositories/eclipse-staging/org/eclipse/jdt/org.eclipse.jdt.core/
Please use latest version from there.